common/changes/@rushstack/ts-command-line/user-danade-AmbiguousParameters_2023-06-12-18-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/ts-command-line",
+      "comment": "Add support for handling ambiguous parameters when conflicting parameters are provided but they provide a non-conflicting alternative (e.g. parameters with the same short-name but different long-names, scoped parameters with the same long-name but different scopes). When using an ambiguous parameter on the CLI, an error message describing the ambiguous parameter usage will appear.",
```

Comment:
Where will the error message appear? Will it be printed to the console?

Reply:
The error message will only appear when an ambiguous command is provided on the CLI. It's printed to the console and the action exits before executing

Reply:
Additionally, the ambiguous parameter no longer appears in the help text. Now only the non-ambiguous options appear there.